tests/include-skip-operation-based/src/test/kotlin/IncludeTest.kt

```diff
@@ -25,7 +26,7 @@ class IncludeTest {
   fun includeVariableTrue() = runBlocking {
     val operation = GetCatIncludeVariableQuery(withCat = true)
 
-    val data = buildQuery {
+    val data = GlobalBuilder.buildQuery {
```

Comment:
This is slightly less ergonomic but it's the price to pay to allow runtime adapters if needed:

```kotlin
Builder(customScalarAdapters).buildQuery {
  ...
}
```